Bottom: 3594d6b9c71842b7297ab8e49baf648b80e715dd
Top:    7be6abbf36c52cd866cef3e4615e3f104b925765
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-01-18 14:56:37 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 485a252f..1e4f56d 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1169,6 +1169,7 @@ static void tun_rx_batched(struct tun_struct *tun, struct tun_file *tfile,
 	if (rcv) {
 		struct sk_buff *nskb;
 		local_bh_disable();
+
 		while ((nskb = __skb_dequeue(&process_queue)))
 			netif_receive_skb(nskb);
 		netif_receive_skb(skb);
